.kitchen.dokken.yml

```diff
@@ -89,7 +89,11 @@ suites:
       - name: 2.1.6
         environment:
           CONFIGURE_OPTS: --disable-install-rdoc
-      - name: 2.2.2
+      - name: 2.4.1
```

Comment:
Just saw in the commit history that this wasn't because of a regression but that it used to be Ruby Enterprise Edition and MRI. Maybe we can drop compiling 2.1.6? It just takes up time and I don't see the additional benefit.